pkg/ocm/client.go

```diff
@@ -124,6 +124,10 @@ func (b *ClientBuilder) Build() (result *Client, err error) {
 	if b.cfg.URL != "" {
 		builder.URL(b.cfg.URL)
 	}
+	// when running ROSA against local CS, redirect AMS API calls to integration
+	if strings.Contains(b.cfg.URL, "localhost") {
+		builder.AlternativeURL("/api/accounts_mgmt", URLAliases["integration"])
```

Comment:
can we remove this . this is a dev config

Reply:
I put this on purpose; w/o this line, running the local ROSA against the local CS is not possible.

Reply:
I think we can document and take it out from the repo . wdyt?

Reply:
That makes sense. I'll remove it.